.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug Report 🐞
+about: Something isn't working as expected? Here is the right place to report.
+title: '[Bug] <title>'
+labels: Bug, Needs Triage
+assignees: ''
+---
+
+<!--
+  To make it easier for us to help you — please follow the suggested format below.
+
+  Before opening a new issue, please search existing issues.
+-->
+
+# Bug Report
+
+## Relevant information
+
+<!-- Provide as much useful information as you can -->
+
+### Your Environment
+
+- Browser:
```

Comment:
Do you find that browser plays a large role these days? I find that the relevant question perhaps is do you use Microsoft browsers 😆 

In reality, we might find a lot of consuming Cypress test suites are running in a headless browser, node env, etc. So I wonder how the environmental information questions might need to accomodate for that.

Reply:
There are still minor differences between Chromium based browsers and Firefox. The cookie issue with larger cookies was not a problem in Firefox, but in Chrome and Edge when I tested it prior to implementing #44. So even for Cypress tests there can be a difference because they can choose Firefox, Chrome, Edge and Electron.